storage/integration_test.go

```diff
@@ -5649,10 +5658,7 @@ func deleteObjectIfExists(o *ObjectHandle, retryOpts ...RetryOption) error {
 	return nil
 }
 
-func writeObject(ctx context.Context, obj *ObjectHandle, contentType string, contents []byte) error {
```

Comment:
Given this is an internal helper func for tests, can you add `forcedEmptyContentType` as a parameter to writeObject instead?

Reply:
We could. I was just trying to avoid having to update many call sites with this.